### PR TITLE
#26 | added reelbox background

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: pixiv6/src/scenes/subscenes/BetPanelScene.ts [hunk 1]
```diff
@@ -26,6 +26,7 @@ export default class BetPanelScene extends BaseScene {
     private layoutManager: LayoutManager = <any>{};
     private betId: number = 1;
     private spinBtnControl: SpinBtnControl = new SpinBtnControl();
+    private isSpinning = false;
```
Comment (hlibSigma) [inline pixiv6/src/scenes/subscenes/BetPanelScene.ts]: Why do we need this bool when we cann't spin while spin btn grey/disabled?

**Review by hlibSigma — CHANGES_REQUESTED** [review]

File: pixiv6/src/scenes/subscenes/BetPanelScene.ts (hunk 1 above)
Comment (ghost) [reply pixiv6/src/scenes/subscenes/BetPanelScene.ts]: it's for dev tool

Comment (ghost) [reply pixiv6/src/scenes/subscenes/BetPanelScene.ts]: removed this variable

**Review by hlibSigma — APPROVED** [review]
review is ok

**Pull request merged**